### ISSUE TITLE
Example?

### ISSUE THREAD

**Issue by trusktr** [perak/dagre-d3#1]
Have a nice example of a graph that updates/morphs as data changes? That'd be cool to see!

**Comment by perak** [comment]
I agree, but I'm super-busy with other things. The best I can now is to add "help wanted" in README.md

**Comment by perak** [comment]
(just updated README)
